syntax.php

```diff
@@ -96,16 +83,22 @@ function handle($match, $state, $pos, Doku_Handler $handler) {
      */
     function render($mode, Doku_Renderer $renderer, $data) {
         global $lang;
+	global $ID;
+	global $INFO;
 
-        if ($mode == 'xhtml') {
+	if ($mode == 'xhtml') {
             $renderer->info['cache'] = false;
 
             @require_once(DOKU_INC.'inc/fulltext.php');
-            $backlinks = ft_backlinks($data[0]);
+	    $page_id = $data[0];
+	    if($page_id == '.') {
+		$page_id = $INFO['id'];
+	    }
+            $backlinks = ft_backlinks($page_id);
 
-            dbglog($backlinks, "backlinks: all backlinks to: $data[0]");
+            dbglog($backlinks, "backlinks: all backlinks to: $page_id");
```

Comment:
why? $page_id is defined and initialized above and makes sure anyone can understand without need to check what is in data and what it means